packages/agent/src/agents/core/snakAgent.ts

```
@@ -276,6 +335,204 @@ export class SnakAgent extends BaseAgent implements IModelAgent {
     return !(!request || request.trim() === '');
   }
 
+  public async *executeAsyncGenerator(
+    input: BaseMessage[] | any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)